clients.go

```diff
@@ -104,7 +104,7 @@ func (c *fasthttpClient) do() (
 	} else {
 		code = resp.StatusCode()
 	}
-	msTaken = uint64(time.Since(start).Nanoseconds() / 1000)
+    usTaken = uint64(time.Since(start).Nanoseconds() / 1000)
```

Comment:
It seems like there's something wrong with indentation here. Have you formatted your code with `gofmt` before committing? 🤔

Reply:
resolved.